docs/recipes/build/bundle-app.md

```diff
@@ -8,27 +8,30 @@ When developing your SAFE application, the local runtime experience uses WebPack
 If you created your SAFE app using the recommended defaults, your application already has a FAKE script which will do the bundling for you. You can create a bundle using the following command:
 
 ```cmd
-dotnet fake build
+dotnet run Bundle
```

Comment:
Do you not need `--` e.g. `dotnet run -- Bundle` or can you just do as is above? If so, that's even better.

Reply:
Nope it just ignores the --